easier_net/sier_net.py

```diff
@@ -277,7 +287,10 @@ def fit(
             if not did_step:
                 break
 
+        return self #added for .fit
+
     def score(self, x: np.ndarray, y: np.ndarray) -> float:
+        self.n_inputs = x.shape[1]
```

Comment:
don't update the model is `score`, because this code should only be evaluating test error